pkg/os/disk/api.go

```diff
@@ -145,7 +145,7 @@ func (DiskAPI) InitializeDisk(diskNumber uint32) error {
 	cmd := fmt.Sprintf("Initialize-Disk -Number %d -PartitionStyle GPT", diskNumber)
 	out, err := utils.RunPowershellCmd(cmd)
 	if err != nil {
-		return fmt.Errorf("error initializing disk %d: %v, %v", diskNumber, out, err)
+		return fmt.Errorf("error initializing disk %d: %v, %v", diskNumber, string(out), err)
```

Comment:
Thanks for making this change.
Without this fix, the error in the driver log is in a byte array.

For example, 

```
{"level":"error","time":"2022-10-05T09:15:20.3428289-07:00","caller":"osutils/windows_os_utils.go:121","msg":"error mounting volume. Parameters: {d106863c-a4a6-4b52-bcec-855f93d0b516 ext4 \\var\\lib\\kubelet\\plugins\\kubernetes.io\\csi\\pv\\pvc-23788f63-fc0d-49f0-b4d1-5d5e440c245d\\globalmount [] false} err: rpc error: code = Unknown desc = error initializing disk 5: [73 110 105 116 105 97 108 105 122 101 45 68 105 115 107 32 58 32 84 104 101 32 100 105 115 107 32 105 115 32 110 111 116 32 108 97 114 103 101 32 101 110 111 117 103 104 32 116 111 32 115 117 112 112 111 114 116 32 97 32 71 80 84 32 112 97 114 116 105 116 105 111 110 32 115 116 121 108 101 46 13 10 65 99 116 105 118 105 116 121 32 73 68 58 32 123 100 53 102 52 52 99 97 97 45 48 48 99 48 45 52 57 49 56 45 56 99 98 101 45 102 50 56 99 56 54 51 100 51 57 101 102 125 13 10 65 116 32 108 105 110 101 58 49 32 99 104 97 114 58 49 13 10 43 32 73 110 105 116 105 97 108 105 122 101 45 68 105 115 107 32 45 78 117 109 98 101 114 32 53 32 45 80 97 114 116 105 116 105 111 110 83 116 121 108 101 32 71 80 84 13 10 43 32 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 126 13 10 32 32 32 32 43 32 67 97 116 101 103 111 114 121 73 110 102 111 32 32 32 32 32 32 32 32 32 32 58 32 78 111 116 83 112 101 99 105 102 105 101 100 58 32 40 83 116 111 114 97 103 101 87 77 73 58 82 79 79 84 47 77 105 99 114 111 115 111 102 116 47 87 105 110 100 111 119 115 47 83 116 111 114 97 103 101 47 77 83 70 84 95 68 105 115 107 41 32 91 73 110 105 116 105 97 108 105 122 101 45 68 105 115 107 93 44 32 67 32 13 10 32 32 32 105 109 69 120 99 101 112 116 105 111 110 13 10 32 32 32 32 43 32 70 117 108 108 121 81 117 97 108 105 102 105 101 100 69 114 114 111 114 73 100 32 58 32 83 116 111 114 97 103 101 87 77 73 32 52 49 48 49 52 44 73 110 105 116 105 97 108 105 122 101 45 68 105 115 107 13 10 32 13 10], exit status 1","TraceId":"927e6863-2009-49f9-92df-cfabe2901e42","stacktrace":"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/osutils.(*OsUtils).NodeStageBlockVolume\n\t/build/pkg/csi/service/osutils/windows_os_utils.go:121\nsigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service.(*vsphereCSIDriver).NodeStageVolume\n\t/build/pkg/csi/service/node.go:103\ngithub.com/container-storage-interface/spec/lib/go/csi._Node_NodeStageVolume_Handler\n\t/go/pkg/mod/github.com/container-storage-interface/[spec@v1.5.0](mailto:spec@v1.5.0)/lib/go/csi/csi.pb.go:6103\ngoogle.golang.org/grpc.(*Server).processUnaryRPC\n\t/go/pkg/mod/google.golang.org/[grpc@v1.40.0](mailto:grpc@v1.40.0)/server.go:1297\ngoogle.golang.org/grpc.(*Server).handleStream\n\t/go/pkg/mod/google.golang.org/[grpc@v1.40.0](mailto:grpc@v1.40.0)/server.go:1626\ngoogle.golang.org/grpc.(*Server).serveStreams.func1.2\n\t/go/pkg/mod/google.golang.org/[grpc@v1.40.0](mailto:grpc@v1.40.0)/server.go:941"}
```

@jingxu97 can we get this merged?